README.md

```diff
@@ -98,29 +98,38 @@ curl -L https://raw.githubusercontent.com/knative/serving/v0.5.2/third_party/ist
 install knative
 
 ```bash
-kubectl apply \
-  --selector knative.dev/crd-install=true \
-  --filename https://github.com/knative/serving/releases/download/v0.5.2/serving.yaml \
-  --filename https://github.com/knative/build/releases/download/v0.5.0/build.yaml \
-  --filename https://github.com/knative/serving/releases/download/v0.5.2/monitoring.yaml \
-  --filename https://raw.githubusercontent.com/knative/serving/v0.5.2/third_party/config/build/clusterrole.yaml
+kubectl apply --selector knative.dev/crd-install=true \
```

Comment:
I did not try out with `Minikube`, but for `GKE` this works with version `0.6.1`

```bash
kubectl apply --selector knative.dev/crd-install=true \
--filename https://github.com/knative/serving/releases/download/v0.6.1/serving.yaml \
--filename https://github.com/knative/build/releases/download/v0.6.0/build.yaml \
--filename https://github.com/knative/eventing/releases/download/v0.6.1/release.yaml \
--filename https://github.com/knative/eventing-sources/releases/download/v0.6.0/eventing-sources.yaml \
--filename https://github.com/knative/serving/releases/download/v0.6.1/monitoring.yaml \
--filename https://raw.githubusercontent.com/knative/serving/v0.6.1/third_party/config/build/clusterrole.yaml
```

```bash
kubectl apply --filename https://github.com/knative/serving/releases/download/v0.6.1/serving.yaml --selector networking.knative.dev/certificate-provider!=cert-manager \
--filename https://github.com/knative/build/releases/download/v0.6.0/build.yaml \
--filename https://github.com/knative/eventing/releases/download/v0.6.1/release.yaml \
--filename https://github.com/knative/eventing-sources/releases/download/v0.6.0/eventing-sources.yaml \
--filename https://github.com/knative/serving/releases/download/v0.6.1/monitoring.yaml \
--filename https://raw.githubusercontent.com/knative/serving/v0.6.1/third_party/config/build/clusterrole.yaml
```

Since we use the version `0.6.1` in `Gopkg.toml`, we should also use the CRDS with the same version

Reply:
right!

Reply:
README have been modified.